helm/prometheus-operator-app/templates/prometheus-operator/service.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ template "kube-prometheus-stack.namespace" . }}
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-operator
+    giantswarm.io/monitoring: "true"
```

Comment:
Can we instead try and use the `ServiceMonitor` this should be enough right? And it should be activated by defaut https://github.com/giantswarm/prometheus-operator-app/blob/7b3ae7876d9e6863cd2881d985fda97ceade8ec9/helm/prometheus-operator-app/values.yaml#L358

Reply:
It's for workload clusters so no :(